### DISCUSSION TITLE
Interpolating first or Upscale first?

### DISCUSSION THREAD

**Discussion by Neo166** [mafiosnik777/enhancr#28] [General]
Hi Team,
I would appreciate some advice. Would you recommend interpolation and then upscale OR vice versa? I am upscaling old 480p anime to 4K 47.8 fps.

I found at times (speculation), that if you interpolate first and then upscale - it takes lesser time than vice versa.

Also, what is your opinion on TTA? Does it really make much difference in Anime encoding?

Does Tiling have any performance improvement for CPU-based encoding?

Regards

**Comment by mafiosnik777** [comment]
Interpolation gets worse with higher resolution, that's why I wouldn't upscale first.

TTA/Ensemble is only for RIFE and yeah it improves quality at cost of speed.

Tiling only has benefits to decrease VRAM and makes everything slower overall.
